app/models/reservation.rb

```diff
@@ -103,6 +103,25 @@ class Reservation < ActiveRecord::Base
       .select('reservations.*, equipment_models.category_id as category_id')
   }
 
+  # for status modifying jobs
+  def self.deletable_missed
```

Comment:
actually, I'm not sure that they should be query objects because they are pretty much just aliases for scope chains